packages/gatsby/cache-dir/api-ssr-docs.js

```diff
@@ -98,3 +93,51 @@ exports.replaceStaticRouterComponent = true
  * }
  */
 exports.onRenderBody = true
+
+/**
+ * Allow a plugin to wrap the page component.
+ *
+ * This is useful for setting wrapper component around pages that won't get
+ * unmounted on page change. For setting Provider components use (#wrapRootComponent)[wrapRootComponent].
+ *
+ * _Note:_ [There is equivalent hook in Browser API](/docs/browser-apis/#wrapPageComponent)
+ * @param {object} $0
+ * @param {object} $0.component The "Page" React Element built by Gatsby.
```

Comment:
I think `component` could be `children`, because its not a component itself but actually ReactNode's
And it would use the same name as in React

Reply:
Right, perhaps `element`? Which would be most accurate as react element is instance of react component (if we follow `React.createElement` or `React.isValidElement` function naming). This also question names of API hooks - should they rather be `wrapPageElement` instead of `wrapPageComponent`?

Reply:
Element would be ok i guess.
Naming things is perhaps the most difficult part of coding :joy:

Reply:
For reference - components vs elements - https://reactjs.org/blog/2015/12/18/react-components-elements-and-instances.html#elements-describe-the-tree